index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"/>
+    <title>Grow Megha Portfolio</title>
+  </head>
+
+  <body>
+    <header>
+      <nav class="header">
+        <ul class="nav-items">
+          <li class="logo"><a>Megha</a></li>
+          <li class="menu"><i class="fa-solid fa-bars"></i></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="main-section">
+      <div class="FirstShape">
+        <img src="/images/Ellipse8.png" alt="shapes"/>
+      </div>
+      <div class="first">
+        <img class="half-ellipse" src="/images/Ellipse11.png" alt="shapes"/>
+      </div>
+      <div>
+        <img class="Ellipse5" src="/images/Ellipse5.png" alt="shapes"/>
+      </div>
+      <div class="tiltsmallrect">
+        <img src="/images/Rectangle7.png" alt="shapes"/>
+      </div>
+      <div>
+        <img class="rectangle4" src="/images/Rectangle4.png" alt="shapes"/>
+      </div>
+      <div>
+        <img class="ellipse3" src="/images/Ellipse3.png" alt="shapes"/>
+      </div>
+    </section>
+
+    <section class="content">
+      <p class="boldtext">
+        I’m Megha<br />
+        Glad to see <br />
+        you!
+      </p>
+      <p class="intro">
+        I’m a software developer! I can help you<br />
+        build a product , feature or website Look <br />
+        through some of my work <br />
+        and experience! If you like what you see and<br />
+        have a project you need coded, don’t <br />
+        hestiate to contact me.
+      </p>
+      <img class="Ellipse12" src="/images/Ellipse12.png" alt="shapes"/>
+      <img class="Ellipse13" src="/images/Ellipse13.png" alt="shapes"/>
+      <p class="socialmediatag">LET'S CONNECT</p>
+      <section>
+        <ul class="socialmediaicons">
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter fa-2x"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-linkedin-in fa-2x"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-medium fa-2x"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-github fa-2x"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fab fa-angellist fa-2x"></i></a>
+          </li>
+        </ul>
+      </section>
+      <img class="Ellipse15" src="/images/Ellipse15.png" alt="shapes"/>
+      <img class="Ellipse4" src="/images/Ellipse4.png" alt="shapes"/>
+      <img class="rectangle5" src="/images/Rectangle5.png" alt="shapes"/>
+    </section>
```

Comment:
- [ ] Please use CSS background images to show these images here. Don't add decoration background images in the HTML.